src/Features/Directory.Build.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Import Project="..\..\Directory.Build.targets"/>
```

Comment:
❕  Use the pattern from **src/Features/Directory.Build.props** to ensure the chain is always correctly sequenced.